dataclients/kubernetes/routegroup.go

```diff
@@ -437,7 +441,8 @@ func explicitGroupRoutes(ctx *routeGroupContext) ([]*eskip.Route, error) {
 					backend:    be,
 				})
 				if err != nil {
-					return nil, err
+					ctx.logger.Errorf("Ignoring route %d: %v", routeIndex, err)
+					continue nextRoute
```

Comment:
Nice to know!

Reply:
In case of /split this will drop routes that might work before

Reply:
No, before it would ignore the whole route group if any of the services is missing.

Reply:
I mean /split with only my-app2 and then you add my-app1 with a bogus config as an example.